pextech/MyRezume
# ft: landing-page

What does this PR do?

-This PR is a responsive version of my landing page.
-This landing page has multiple sections where users can dive in.
 
How should this be manualy be tested?

-check this repo with Git clone https://github.com/pextech/MyRezume.git
-checkout for ft-landing-page

Any background context you want to provide

N/A

What are the relevant trello stories?
![landing](https://user-images.githubusercontent.com/57200088/97723654-b3fca500-1ad4-11eb-8242-42ce1bd9ba07.PNG)
![land-mobile](https://user-images.githubusercontent.com/57200088/97723682-ba8b1c80-1ad4-11eb-8308-0f94222fe7d0.PNG)

https://trello.com/c/jVZxSwGU
